src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -15,23 +15,13 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <PackageId>Microsoft.OpenApi.OData</PackageId>
     <SignAssembly>true</SignAssembly>
-    <Version>1.1.0</Version>
+    <Version>1.2.0</Version>
```

Comment:
should we use a preview suffix here?

Reply:
If it is going out with this hot-fix [PR](https://github.com/microsoft/OpenAPI.NET.OData/pull/288), then I think we should.

Reply:
even if it's not a hotfix, my worry is that if we bump the minor so early in the process of producing it, we might "release it" by mistake. Using a preview suffix prevents us from accidently locking this version, and also allows people to preview the changes.